src/stories/organisms-plan.stories.js

```diff
@@ -0,0 +1,6 @@
+import Item_Plan from "../components/organisms/plan/plan.html";
+import "../components/organisms/plan/plan.scss";
+
+export default { title: "Organisms/Plan" };
+
+export const Events = () => Item_Plan;
```

Comment:
```suggestion
export const Plans = () => Item_Plan;
```
![image](https://user-images.githubusercontent.com/83479175/187020362-08206f93-c874-4440-9bfc-d143a0b2b810.png)
planなのかeventsなのかわからない感じなので、plan に修正をお願いします。